### PR TITLE
[owners] Update README for owners

### PR DESCRIPTION
**Pull request by rcebulko**
Also adds npm commands to initialize the cache and run the info/dev server

### CODE REVIEW

File: owners/package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "node": "^10.11"
   },
   "scripts": {
-    "dev": "nodemon",
```
Comment (danielrozenberg) [inline owners/package.json]: If you're removing the use of nodemon, maybe remove nodemon as a dep?

File: owners/scripts/warm_cache.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const {
   GITHUB_ACCESS_TOKEN,
   GITHUB_OWNER,
   GITHUB_REPOSITORY,
-} = process.env.CLOUD_STORAGE_BUCKET;
```
Comment (danielrozenberg) [inline owners/scripts/warm_cache.js]: 😬

Comment (rcebulko) [reply owners/scripts/warm_cache.js]: Lol yep, not sure how that go there :eyes:

File: owners/package.json (hunk 1 above)
Comment (rcebulko) [reply owners/package.json]: Done!

**Pull request merged**